### PR TITLE
added a documentation for webrequest 

### LINKED ISSUE #66: Add docs on making web requests
Add simple docs on making web requests akin to this:

https://github.com/SAFE-Stack/SAFE-BookStore/blob/master/src/Client/pages/Login.fs#L57

### PR DESCRIPTION
**Pull request by SergejDK**
Documentation for making a webrequest is added in the documentation.

Is there a need for the example application?

fix #66

### CODE REVIEW

**Comment by TimLariviere** [comment]
Thanks! It looks good.
Indentation could use a little fixing, but apart from that, it's great.

>Is there a need for the example application?

Yes, please.

**Comment by SergejDK** [comment]
Will take a look at the identation this evening. 
Sample will be added in the samples folder, too.

**Comment by SergejDK** [comment]
MacOs, iOS and GTK need to be tested. 
Somehow i can't add a nuget package there...

**Comment by SergejDK** [comment]
Did not change anything, now it works on iOS, macOS and GTK.
But macOS has still a bug. When clicked on "MainPage" it gets a nullreference exception.

**Comment by TimLariviere** [comment]
@SergejDK Yes, I saw this one when I tested the macOS node of AllControls.
If you can gather more information and open an issue, it would be great.
Thank you.

I'll take a look later to see if I can find why you were having an error with FSharpValueOption.

**Comment by SergejDK** [comment]
The issue with the nullreference is in XF itself. 
Will open an issue here just to show the others that is in progress.

**Comment by SergejDK** [comment]
The Travis build fails because it can not download fsharp data into the Android project. Any ideas ?

**Comment by TimLariviere** [comment]
Yes, the samples don't use NuGet/packages.config.
Instead, we add the dependency in paket (https://github.com/fsprojects/Fabulous/blob/master/paket.dependencies) and then we make a manual reference to the dll in the fsproj.

This is kinda counter-intuitive.
But someday, I hope we can make that better. See #181 and #222.

For example: FSharp.Core in Android
https://github.com/fsprojects/Fabulous/blob/0f76787cba92635a6b7cb2c90242c5975b6b9bbc/paket.dependencies#L17-L20

https://github.com/fsprojects/Fabulous/blob/0f76787cba92635a6b7cb2c90242c5975b6b9bbc/samples/AllControls/Droid/AllControls.Droid.fsproj#L167-L169

So, to add FSharp.Data for Android, you can add a line in the group `androidapp` in the paket.dependencies file.
Make sure to do a `paket.exe install` to download the dependency on your machine.
Then change the path to the dll in the fsproj.

Also remove the unused packages.config.

**Comment by SergejDK** [comment]
Thanks! 
Forgot about Paket ... 
Will change asap

File: samples/AllControls/Droid/AllControls.Droid.fsproj [hunk 1]
```diff
@@ -163,10 +164,13 @@
     </Reference>
     <Reference Include="Xamarin.Forms.Xaml">
       <HintPath>../../../packages/androidapp/Xamarin.Forms/lib/MonoAndroid10/Xamarin.Forms.Xaml.dll</HintPath>
-    </Reference>
+    </Reference> 
     <Reference Include="FSharp.Core">
       <HintPath>../../../packages/androidapp/FSharp.Core/lib/netstandard1.6/FSharp.Core.dll</HintPath>
     </Reference>
+    <Reference Include="FSharp.Data">
+      <HintPath>../../../packages/androidapp/FSharp.Data.3.0.0/lib/netstandard2.0/FSharp.Data.dll</HintPath>
```
Comment (TimLariviere) [inline samples/AllControls/Droid/AllControls.Droid.fsproj]: Paket doesn't add the version number in the path
```suggestion
      <HintPath>../../../packages/androidapp/FSharp.Data/lib/netstandard2.0/FSharp.Data.dll</HintPath>
```

File: samples/AllControls/Droid/AllControls.Droid.fsproj [hunk 2]
```diff
@@ -78,6 +78,7 @@
     <AndroidResource Include="Resources\drawable-xhdpi\icon.png" />
     <AndroidResource Include="Resources\drawable-xxhdpi\icon.png" />
     <None Include="Assets\AboutAssets.txt" />
+    <Content Include="packages.config" />
```
Comment (TimLariviere) [inline samples/AllControls/Droid/AllControls.Droid.fsproj]: Can you remove this line and the file? Thanks

File: samples/AllControls/AllControls/AllControls.fsproj [hunk 3]
```diff
@@ -8,6 +8,7 @@
       <EmbeddedResource Include="Baboon_Serengeti.jpg" />
       <EmbeddedResource Include="styles.css" />
       <PackageReference Include="FSharp.Core" />
+      <PackageReference Include="FSharp.Data" Version="3.0.0" />
```
Comment (TimLariviere) [inline samples/AllControls/AllControls/AllControls.fsproj]: Versioning is centralized in the `Directory.Build.props` file at the root of the repository.
In this file, you only need to include the package (without version)
Then in the `Directory.Build.props`, you can add the version, just like the other packages.
```suggestion
      <PackageReference Include="FSharp.Data" />
```

**Comment by TimLariviere** [comment]
@SergejDK The sample works great!

**Comment by TimLariviere** [comment]
Thank you. I'm merging it now.

**Comment by dsyme** [comment]
This is a fantastic addition to the docs

**Pull request merged**